homeassistant/components/alexa/capabilities.py

```diff
@@ -1435,6 +1447,12 @@ def get_property(self, name: str) -> Any:
             if mode in modes:
                 return f"{humidifier.ATTR_MODE}.{mode}"
 
+        # Remote Activity
+        if self.instance == f"{remote.DOMAIN}.{remote.ATTR_ACTIVITY}":
+            mode = self.entity.attributes.get(remote.ATTR_CURRENT_ACTIVITY, None)
+            if mode in self.entity.attributes.get(remote.ATTR_ACTIVITY_LIST, None):
+                return f"{remote.ATTR_ACTIVITY}.{mode}"
+
```

Comment:
Let's stick to the English locale, in this PR.

Reply:
I reverted the commit which uses the locale from the config. Now every mode is using the default locale 'en-US'.